src/test/ui/use/use-mod.stderr

```diff
@@ -20,13 +20,12 @@ LL |     self,
    |     ---- previous import of the module `bar` here
 ...
 LL |     self
-   |     ^^^^ `bar` reimported here
+   |     ^^^^
+   |     |
+   |     `bar` reimported here
+   |     help: remove unnecessary import
```

Comment:
can you add another case for use `use {self, self, }`? Want to verify that the resulting suggestion doesn't end up with `use {self, , }`.

Reply:
I see that there are already some cases for this, disregard.